cloud/azure/management/recovery/mode/backupjobsstatus.pm

```diff
@@ -140,6 +140,8 @@ sub check_options {
     }
 
     $self->change_macros(macros => ['warning_status', 'critical_status']);
```

Comment:
This change could be a good opportunity to refactor the code and replace catalog_status_threshold by catalog_status_threshold_ng. 

You will get rid of this, and also of the change_macros line 142. 

Example in my current draft PR (https://github.com/centreon/centreon-plugins/pull/3804/files#diff-9a6e78030abae4ae7951b890d4fda32b4bd523e174eefcb6740ed7367a27ab73)